avm/utilities/pipelines/e2eValidation/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -94,6 +94,7 @@ function Initialize-DeploymentRemoval {
             'Microsoft.ContainerInstance/containerGroups' # Must be removed before their MSI
             'Microsoft.ManagedIdentity/userAssignedIdentities',
             'Microsoft.Databricks/workspaces'
+            'Microsoft.DevOpsInfrastructure/pools' # Must be removed before other resources it depends on like a virtual network
```

Comment:
Before merging we should temporarily disable the actions. No need to run al workflows for this change

Reply:
Approved. Merge responsibly.